composite/bitcoin-json-rpc/src/adapter.ts

```diff
@@ -0,0 +1,44 @@
+import JSONRPC from '@chainlink/json-rpc-adapter'
+import { Execute, AdapterRequest } from '@chainlink/types'
```

Comment:
In general I would recommend you to follow any of the recent PR's. [`dns-record-check`](https://github.com/smartcontractkit/external-adapters-js/pull/199) is very recent and similar. I think it could help

Reply:
True, I had already checked it.